src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -140,13 +142,18 @@ export default class PeopleAndRoles extends Mixins(EntityFilterMixin) {
   readonly Roles = Roles
   readonly IncorporatorTypes = IncorporatorTypes
 
+  mounted (): void {
+    this.setAddPeopleAndRoleStepValidity(this.hasValidRoles())
+  }
+
   // Methods
   private addOrgPerson (rolesToInitialize: string[], type: string): void {
```

Comment:
Can "type" be a type (enum) instead of a string?

Reply:
Good catch here. Applies to both arguments!